src/main/java/com/fcgl/madrid/dev/IndexController.java

```diff
@@ -0,0 +1,52 @@
+package com.fcgl.madrid.dev;
+
+import com.fcgl.madrid.search.dataModel.Customer;
+import com.fcgl.madrid.search.repository.CustomerRepository;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import com.mongodb.MongoClient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+@Controller
+@RequestMapping("/")
+public class IndexController {
+    private final Logger logger = Logger.getLogger(this.getClass().getName());
+
+    @Autowired
+    MongoClient mongoClient;
+    @Autowired
+    private CustomerRepository repository;
+
+    @GetMapping(value= "/test")
```

Comment:
I wanted this stuff to get pre-populated. But there is a small delay when connecting to mongo and it breaks the service. Will have to look into this separately but should not be a blocker. Just annoying to test that you have your collections set up correctly